.bazelci/presubmit.yml

```diff
@@ -6,6 +6,8 @@ tasks:
     platform: ubuntu1604
     run_targets:
     - "@nodejs//:yarn_node_repositories"
+    # Regression test for #1493
```

Comment:
Only a 6s task so I think also add to mac and Windows to make sure we're good on all platforms. I'll make a note to add a more rigorous test on the .pack output.

Reply:
done

I don't think there's much need to test the logic or the result of the pack operation because the code here is just to run `npm pack` - very small surface area for bugs where this run command succeeds but does the wrong thing.